src/createDispatcher.js

```diff
@@ -50,19 +56,27 @@ export default function createDispatcher(opts = {}) {
 
   const logging = parseOpts(opts.logging)
   if (logging.agendas) {
+    // Seems like this means that middlewares are effectively an attachment on dispatchers
     logAgendas(dispatcher)
   }
 
   function next(agenda) {
     const obs = agenda
       .subscribeOn(scheduler)
-      .publishReplay()
+      .publishReplay() // Would this make all my actions replayable?
+                       // What if they are expensive or can't be performed again?
+                       // I definitely need to look up publishReplay when I have internet again
       .refCount()
```

Comment:
In combination with the refcount this stores the agenda's result for when it's used in reduce. Because if the way that reduce works (deep subscribing for each store) we need to make sure that as long as there's some processing going on inside that, we don't lose the result